cloudbuild.yaml

```diff
@@ -0,0 +1,46 @@
+# Reference: https://cloud.google.com/cloud-build/docs/build-config
+
+steps:
+  # Pull pre-existing latest Docker image.
+  - name: 'gcr.io/cloud-builders/docker'
+    id: pull_image
+    waitFor: ['-']
+    timeout: 1h
+    args: ['pull', 'gcr.io/oak-ci/oak:latest']
+  # Build Docker image based on current Dockerfile, if necessary.
+  - name: 'gcr.io/cloud-builders/docker'
+    id: build_image
+    waitFor: ['pull_image']
+    timeout: 1h
+    args: ['build', '--pull', '--cache-from=gcr.io/oak-ci/oak:latest', '--tag=gcr.io/oak-ci/oak:latest', '.']
+  # Run build step inside the newly created Docker image.
+  # See: https://cloud.google.com/cloud-build/docs/create-custom-build-steps
+  - name: 'gcr.io/oak-ci/oak:latest'
+    id: build_code
+    waitFor: ['build_image']
+    entrypoint: 'bash'
+    args: ['./scripts/build_server']
+  # Copy compiled enclave binary to workspace so that it can be uploaded as artifact.
+  - name: 'gcr.io/oak-ci/oak:latest'
+    id: move_artifacts
+    waitFor: ['build_code']
+    entrypoint: 'cp'
+    args: ['./bazel-bin/oak/server/asylo/oak_enclave_unsigned.so', './oak_enclave_unsigned.so']
```

Comment:
Hmm... isn't there a way for bazel to specify the output of the binary instead of copying it?

Reply:
Not AFAICT.